python-toolbox/CHANGES.md

```diff
@@ -1,5 +1,22 @@
 ## Changes log
 
+### 0.0.5
+
+   - Artifact reload bug fix #3 #5
+   - Dependency Links Deprecation & Remove #14
+   - Dependencies version fix for MacOS env #17
+   - General contribution file update #10
+   - CI update
+    - Move Travis CI out from sub-directory #1 #5
```

Comment:
I will suggest having one more SPACE in front of this so it will have a new line.
- a
  - b

Reply:
oh yea, thanks Wei, fixing it.